split_ivp_fvp.sh

```diff
@@ -0,0 +1,4 @@
+head -n 1 $1 > initial_visits.csv
```

Comment:
Make sure you put the shebang line so the kernel knows this is an executable script.
```suggestion
#!/bin/sh
head -n 1 $1 > initial_visits.csv
```